src/c4/ifpstream.hh

```diff
@@ -0,0 +1,54 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/ifpstream.hh
+ * \author Mathew Cleveland
+ * \brief  Define class ifpstream
```

Comment:
Change the brief to say something about being a 'mpi-parallel file reader'.

Reply:
Fixed.

Reply:
Fixed.